### PR TITLE
Restock: Add Restock view and functionality added, restockContext finalizing the api call addrestock to follow

### PR DESCRIPTION
**Pull request by lilianyangc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-Inventory Management Application for Tender Plant and Care.
\ No newline at end of file
+Inventory Management Application for Tender Plant and Care.
+
+wow
```
Comment (AshenCat) [inline README.md:3]: :shibaLaugh:

File: src/components/Forms/AddRestock/AddRestock.js [hunk 2]
```diff
@@ -1,9 +1,282 @@
...
+    onValueChange();
+  }, [unitCost, quantity]);
+
+  useEffect(() => {
+    const onFormCheck = () => {
+      if (product._id && supplier.id && unitCost > 0 && quantity > 0)
+        setValidForm(true);
+    };
+    onFormCheck();
+  }, [product, supplier, unitCost, quantity]);
```
Comment (AshenCat) [inline src/components/Forms/AddRestock/AddRestock.js:95]: Does it really need to check everytime there's an input change? Isn't it better to just check once you submit the form?

Comment (lilianyangc) [reply src/components/Forms/AddRestock/AddRestock.js:103]: Oh, I needed this one for the disabled Confirm button before anyone could submit.

Comment (AshenCat) [reply src/components/Forms/AddRestock/AddRestock.js:103]: Ah, it's good then!

Comment (lilianyangc) [reply src/components/Forms/AddRestock/AddRestock.js:103]: Maybe, there's still another way. I think update this later.

**Pull request merged**